src/partition/repart.c

```diff
@@ -2519,6 +2519,9 @@ static int context_read_definitions(Context *context) {
                 *dest = p;
         }
 
+        if (esp->priority > 0)
+                log_warning("ESP definition has a priority set, which might create un-bootable installs.");
```

Comment:
maybe say "… a non-zero priority set, …"

Reply:
Here too. Not really needed in a more generic solution, so dropping 99fcce8a08c112df5cecc914bc76d163239b2707